code/modules/client/preference_setup/loadout/loadout_factions.dm

```diff
@@ -146,6 +149,8 @@
 	zavodskoi_labcoats["zavodskoi letterman labcoat"] = /obj/item/clothing/suit/storage/toggle/labcoat/zavodskoi/letterman
 	zavodskoi_labcoats["zavodskoi letterman labcoat, alt"] = /obj/item/clothing/suit/storage/toggle/labcoat/zavodskoi/letterman/alt
 	zavodskoi_labcoats["zavodskoi labcoat, long"] = /obj/item/clothing/suit/storage/toggle/longcoat/zavodskoi
+	zavodskoi_labcoats["zavodskoi officer coat"] = /obj/item/clothing/suit/storage/security/officer/zav
+	zavodskoi_labcoats["zavodskoi officer coat, alt"] = /obj/item/clothing/suit/storage/security/officer/zav/alt
```

Comment:
These new coats should be in a new subcategory, that only sec can take. The items should also be named "zavodskoi security officer coat" and not just "zavodskoi officer coat".

Reply:
renamed to security coats because not only security officers use them